src/shapes/text.class.js

```diff
@@ -908,7 +913,7 @@
       var box = {
         width: width,
         left: 0,
-        height: charStyle.fontSize,
+        height: style.fontSize,
         kernedWidth: kernedWidth,
       };
```

Comment:
i think we can just return deltay: deltay in the box object, is more than fine for now.